src/main/java/com/salesforce/dev/framework/DriverManager.java

```diff
@@ -81,6 +90,10 @@ public WebDriverWait getWait(){
 
     }
 
+    public int getTimeoutNormal(){
```

Comment:
I think this method is unnecessary
you can call the static attribute TIMEOUT_NORMAL